tests/v2/validation/extensions/provisioning/audits/audits.go

```diff
@@ -0,0 +1,340 @@
+package audits
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"strings"
+	"testing"
+	"time"
+
+	provv1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	rkev1 "github.com/rancher/rancher/pkg/apis/rke.cattle.io/v1"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	steveV1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters/bundledclusters"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeconfig"
+	nodestat "github.com/rancher/rancher/tests/framework/extensions/nodes"
+	"github.com/rancher/rancher/tests/framework/extensions/provisioning"
+	psadeploy "github.com/rancher/rancher/tests/framework/extensions/psact"
+	"github.com/rancher/rancher/tests/framework/extensions/registries"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads/pods"
+	"github.com/rancher/rancher/tests/framework/pkg/wait"
+	"github.com/rancher/rancher/tests/v2prov/defaults"
+	wranglername "github.com/rancher/wrangler/pkg/name"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	appv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apimachinery/pkg/watch"
+	capi "sigs.k8s.io/cluster-api/api/v1beta1"
+)
+
+const (
+	namespace                   = "fleet-default"
+	kwait5                      = 5
+	logMessageKubernetesVersion = "Validating the current version is the upgraded one"
+)
+
+// AuditRKE1Cluster validates that the RKE1 cluster and its resources are in a good state, matching a given config.
+func AuditRKE1Cluster(t *testing.T, client *rancher.Client, clustersConfig *provisioning.ClusterConfig, clusterName string) {
```

Comment:
I 100% agree, I think the provisioning package name is enough to distinguish this. And using the cluster object instead of cluster names would reduce the repetitive snippets significantly.